internal/cmd/egctl/envoy_dashboard.go

```diff
@@ -0,0 +1,134 @@
+// Copyright Envoy Gateway Authors
+// SPDX-License-Identifier: Apache-2.0
+// The full text of the Apache license is available in the LICENSE file at
+// the root of the repo.
+
+package egctl
+
+import (
+	"errors"
+	"fmt"
+	"io"
+	"os"
+	"os/exec"
+	"os/signal"
+	"runtime"
+
+	"github.com/spf13/cobra"
+	"k8s.io/apimachinery/pkg/types"
+
+	kube "github.com/envoyproxy/gateway/internal/kubernetes"
+)
+
+func newEnvoyDashboardCmd() *cobra.Command {
+	var podName, podNamespace string
+	var listenPort int
+
+	dashboardCmd := &cobra.Command{
+		Use:   "envoy-proxy <name> -n <namespace>",
+		Short: "Retrieves Envoy admin dashboard for the specified pod",
+		Long:  `Retrieve Envoy admin dashboard for the specified pod.`,
+		Example: `  # Retrieve Envoy admin dashboard for the specified pod.
+  egctl experimental dashboard envoy-proxy <pod-name> -n <namespace>
+
+  # short syntax
+  egctl experimental d envoy-proxy <pod-name> -n <namespace>
```

Comment:
can you add a section in https://github.com/envoyproxy/gateway/blob/main/site/content/en/latest/user/egctl.md to describe more about this subcommand?